ggshield/cmd/sca/scan/sca_scan_utils.py

```diff
@@ -14,27 +21,22 @@
 from ggshield.core.scan.scan_mode import ScanMode
 from ggshield.core.tar_utils import INDEX_REF, get_empty_tar, tar_from_ref_and_filepaths
 from ggshield.core.text_utils import display_error, display_info
-from ggshield.verticals.sca.client import SCAClient
 from ggshield.verticals.sca.file_selection import (
     get_all_files_from_sca_paths,
     sca_files_from_git_repo,
 )
 from ggshield.verticals.sca.output.handler import SCAOutputHandler
 from ggshield.verticals.sca.output.json_handler import SCAJsonOutputHandler
 from ggshield.verticals.sca.output.text_handler import SCATextOutputHandler
-from ggshield.verticals.sca.sca_scan_models import (
-    ComputeSCAFilesResult,
-    SCAScanAllOutput,
-    SCAScanDiffOutput,
-    SCAScanParameters,
-)
 
 
 def get_scan_params_from_config(sca_config: SCAConfig) -> SCAScanParameters:
     return SCAScanParameters(
         minimum_severity=sca_config.minimum_severity,
         ignored_vulnerabilities=[
-            ignored_vuln
+            SCAIgnoredVulnerability(
```

Comment:
I noticed that the code here transforms SCAIgnoredVulnerability (from ggshield) instances into SCAIgnoredVulnerability (from pygitguardian) instances. What is the purpose of having two separate classes ? If there is one shouldn't we avoid having the same name for both ?

Reply:
And why didn't we do this transformation before ?

Reply:
The one in py-gitguardian is more consise, and used as an intermediate type for `ScanParameters` that are sent to the backend. The one in GGShield is used to parse userconfig, and we did not wanted to add userConfig related information to a model in py-gitguardian. 
The one in GGShield could inherit from the first one if we want to. We perform the transformation here as it is exactly the purpose of the one in py-gitguardian, to be used in `SCAScanParameters`.

Reply:
I can rename the one in the config `SCAConfigIgnoredVulnerability` if you find this confusing.

Reply:
It makes sense like this. I have no strong opinion on whether to rename it or not.